lua/telescope/previewers/buffer_previewer.lua

```diff
@@ -15,6 +15,50 @@ local previewers = {}
 
 local ns_previewer = vim.api.nvim_create_namespace('telescope.previewers')
 
+local color_hash = {
+  ['p'] = 'TelescopePreviewPipe',
+  ['c'] = 'TelescopePreviewCharDev',
+  ['d'] = 'TelescopePreviewDirectory',
+  ['b'] = 'TelescopePreviewBlock',
+  ['l'] = 'TelescopePreviewLink',
+  ['s'] = 'TelescopePreviewSocket',
+  ['.'] = 'TelescopePreviewNormal',
+  ['r'] = 'TelescopePreviewRead',
+  ['w'] = 'TelescopePreviewWrite',
+  ['x'] = 'TelescopePreviewExecute',
+  ['-'] = 'TelescopePreviewHypen',
```

Comment:
shouldn't this be `hyphen` instead of `hypen`?

Reply:
Yeah typo thanks